emulators/krunkit/Portfile

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+# Upstream build source (until more official): https://github.com/slp/homebrew-krunkit/blob/main/Formula/krunkit.rb
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cargo_fetch 1.0
+
+github.setup        containers krunkit 0.1.4 v
+github.tarball_from archive
+
+revision            0
+
+categories          emulators
+license             Apache-2
+maintainers         {@tsujp wz.ht:jc+macports} openmaintainer
+
+description         Tool to launch configurable virtual machines using libkrun
+long_description    {*}${description}.
+
+# Only arm64 as libkrun only supports Hypervisor.framework on arm64.
+supported_archs     arm64
+
+checksums           ${distname}${extract.suffix} \
+                    rmd160  e7d180e5ec342f9672bfe8efa5c0e8453f4d0802 \
+                    sha256  30998931a859b4853adb1d3e66ec5072a92edb9d3e46ad69c41a647584c619a0 \
+                    size    18686
+
+depends_run         port:libkrun
```

Comment:
This also needs to be `depends_lib`, since `krunkit` links against a library installed by `libkrun`:

```
:) cllang@gallumbits:[…]/dports/emulators/krunkit$ otool -L /opt/local/bin/krunkit
/opt/local/bin/krunkit:
	/opt/local/lib/libkrun-efi.dylib (compatibility version 0.0.0, current version 0.0.0)
[…]
```

Reply:
Ah, is this perhaps why I had to do that rpath stuff with the patchfile hmm. Will enact and check (in either case adopting `depends_lib` though).

Reply:
please make the change(s) requested

Reply:
Enacted.